icu4c/source/common/unicode/ubrk.h

```diff
@@ -321,6 +323,24 @@ ubrk_safeClone(
           int32_t *pBufferSize,
           UErrorCode *status);
 
+#endif /* U_HIDE_DEPRECATED_API */
+
+#ifndef U_HIDE_DRAFT_API
+
+/**
+ * Thread safe cloning operation.
+ * @param bi iterator to be cloned
+ * @param status to indicate whether the operation went on smoothly or there were errors
+ *  An informational status value, U_SAFECLONE_ALLOCATED_ERROR, is used for an allocation error.
```

Comment:
Surely this line does not apply to this function. Please remove.

Reply:
Removed. Done.